src/charm.py

```diff
@@ -32,7 +32,6 @@ class PolkadotCharm(ops.CharmBase):
 
     def __init__(self, *args):
         super().__init__(*args)
-        self.prometheus_node_provider = PrometheusProvider(self, 'node-prometheus', 9100, '/metrics')
```

Comment:
What we could have done is to keep the relation, but move the install process to the relation hook. Then the charm would continue to support it. We could add uninstall of the node-exporter to the relation hook as well. Then all we would have to do in the models would be
1. refresh charm
2. remove relation